proto/desmos/profiles/v1beta1/query_relationships.proto

```diff
@@ -29,15 +29,14 @@ message QueryUserRelationshipsRequest {
 // QueryUserRelationshipsResponse is the response type for the
 // Query/UserRelationships RPC method.
 message QueryUserRelationshipsResponse {
```

Comment:
I think this could be now renamed to `QueryRelationshipsResponse`, and the same for `QueryRelationshipsRequest`.